packages/bpk-component-nudger/src/BpkNudger.tsx

```diff
@@ -16,41 +16,79 @@
  * limitations under the License.
  */
 
+import { useRef } from 'react';
+
+import { BpkButtonV2, BUTTON_TYPES } from '../../bpk-component-button';
+import { withButtonAlignment } from '../../bpk-component-icon';
+import MinusIcon from '../../bpk-component-icon/sm/minus';
+import PlusIcon from '../../bpk-component-icon/sm/plus';
 // @ts-expect-error Untyped import. See `decisions/imports-ts-suppressions.md`.
 import BpkLabel from '../../bpk-component-label';
 import BpkText, { TEXT_STYLES } from '../../bpk-component-text';
-import { cssModules } from '../../bpk-react-utils';
+import { cssModules, setNativeValue } from '../../bpk-react-utils';
 
-import BpkConfigurableNudger from './BpkConfigurableNudger';
 import { type CommonProps } from './common-types';
 
 import STYLES from './BpkNudger.module.scss';
 
 const getClassName = cssModules(STYLES);
 const compareValues = (a: number, b: number): number => a - b;
-const incrementValue = (currentValue: number): number => currentValue + 1;
-const decrementValue = (currentValue: number): number => currentValue - 1;
-const formatValue = (currentValue: number): string => currentValue.toString();
 
 const BpkNudger = ({
   buttonType = 'secondary',
   className = null,
+  decreaseButtonLabel,
   icon,
   id,
+  increaseButtonLabel,
+  inputClassName,
+  max,
+  min,
```

Comment:
Wonder if this should be clamped down in the code and be set as a default to 0, so it can never drop below zero as minus numbers never come up for nudgers

Reply:
Could that be to restrictive? I think consumers will most likely not go below 0. We can set a default min of 0?

Reply:
I think it wouldn't be too restrictive, as we would be setting a correct expectation of what the system should be and avoid people possibly putting bad data into our systems and causing errors, so if you try to go beyond 0 it would bring it back in and not accidently get set below

Reply:
Is this what you had in mind - https://github.com/Skyscanner/backpack/pull/3576/commits/954cbd823df21c986270fe831e19ee6d558cb27f ?